install.sql

```diff
@@ -6,6 +6,9 @@ CREATE TABLE IF NOT EXISTS `#__virtuemart_shipment_plg_zasilkovna` (
 	`zasilkovna_packet_id` decimal(10,0),
 	`zasilkovna_packet_price` decimal(15,2),
 	`weight` decimal(10,4),
+	`length` smallint unsigned,
```

Comment:
co spíš `smallint(4) unsigned NULL`?

Reply:
pomocí (4) dáš jasně najevo, jaké hodnoty tam očekáváš, NULL viz ticket

Reply:
ty NULLy, to beru
tu sirku (4) to neberu, je to k nicemu a navic v mysql deprecated

https://dev.mysql.com/doc/refman/8.4/en/numeric-type-attributes.html
cituji:
The ZEROFILL attribute is deprecated for numeric data types, as is the display width attribute for integer data types. You should expect support for ZEROFILL and display widths for integer data types to be removed in a future version of MySQL.